docs/report.md

```diff
@@ -68,12 +68,20 @@ Briefly describe what kinds of tests you have in your test suites and what they
 
 ### License
 
-State which software license you chose for your application.
+None of the libraries used has the GPL license, or any other copyleft license. 
+Therefore the choice of license was open, and the MIT license was used. 
```

Comment:
Therefore the choice of license was open -> Therefore the choice of license was _left_ open

Maybe?

Reply:
And the MIT license was chosen